app/src/main/kotlin/com/intive/picover/parties/viewmodel/PartyDetailsViewModel.kt

```diff
@@ -0,0 +1,29 @@
+package com.intive.picover.parties.viewmodel
+
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.SavedStateHandle
+import com.intive.picover.common.viewmodel.LifecycleViewModel
+import com.intive.picover.parties.repository.MockedPartiesRepository
+import com.intive.picover.parties.state.PartyDetailsState
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class PartyDetailsViewModel @Inject constructor(
+	private val partiesRepository: MockedPartiesRepository,
+	savedStateHandle: SavedStateHandle,
+) : LifecycleViewModel() {
+
+	val partyDetails: MutableState<PartyDetailsState> = mutableStateOf(PartyDetailsState.Loading)
+	private val partyId: Int = savedStateHandle["partyId"]!!
+
+	override fun onFirstCreate() {
+		super.onFirstCreate()
```

Comment:
super.onFirstCreate() can be omitted here as well

Reply:
Done